network-api/networkapi/wagtailpages/templates/wagtailpages/homepage.html

```diff
@@ -28,6 +28,8 @@ <h1 class="h1-heading">{{ page.hero_headline }}</h1>
     </div>
   </div>
 
+  {% include "./fragments/spotlight_posts.html" %}
```

Comment:
Very minor tweak that might make absolutely no difference: we could pass in `posts=page.spotlight_posts.all` into this include

Reply:
As a dedicated fragment, it's nicer to be explicit in the fragment itself, rather than having to hunt for where `posts` came from. If we end up reusing this in other templates, then making it passable would make sense.